lib/game_end_evaluator.rb

```diff
@@ -19,9 +19,9 @@ def any_remaining_moves?(board)
     board.find_available_positions.length.positive?
   end
 
-  def find_winner_in_matrix(matrix)
-    return 'X' if matrix.include? Array.new(3, 'X')
-    return 'O' if matrix.include? Array.new(3, 'O')
+  def find_winner_in_matrix(matrix:, width:)
+    return 'X' if matrix.include? Array.new(width, 'X')
```

Comment:
is it possible to achieve this functionality without using 'magic' `'X', 'O'` strings?

Reply:
I'm having a hard time removing it from `Minimax`. I see that one of the cards is selecting a custom mark for each user. Do you think I could tackle this problem then?